pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -1868,6 +1868,16 @@ def test_invalid_origins_tzinfo(self):
             pd.to_datetime(1, unit='D',
                            origin=datetime(2000, 1, 1, tzinfo=pytz.utc))
 
+    @pytest.mark.parametrize("kwargs", [
```

Comment:
Nit: Instead of `kwargs`, I think I'd be clearer if this test parametrizes over `format = [None, "%Y-%m-%d %H:%M:%S"]`

Reply:
That makes sense.  Done.